monorepo/packages/wallet-connectors/src/index.tsx

```diff
@@ -74,6 +81,8 @@ export { HashConnector, HashConnectEvents, hashconnectEvent, WalletConnectConnec
 export { UserRejectedRequestError, NoEthereumProviderError };
 export type { WalletConnectConnectorArguments };
 
+export { near };
 export * from './NearConnector/near';
```

Comment:
Actually this is not needed, I am not sure why you moved `near` from `near.ts` to `index.ts`. If its justifiable then lets keep it here or else lets revert all these changes.